frontend/schemaCMS/src/shared/components/sourceForm/sourceForm.component.js

```diff
@@ -98,6 +104,22 @@ export class SourceFormComponent extends PureComponent {
     />
   );
 
+  renderGoogleUploader = ({ setFieldValue, fileName, disabled, ...restProps }) => {
```

Comment:
I don't think this is a great name, we don't upload anything to Google. I think `renderSpreadsheetInput` explains it better

Reply:
fixed